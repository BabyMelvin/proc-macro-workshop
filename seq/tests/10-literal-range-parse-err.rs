fn main() {
    let val = seq::seq!(N in -1..1 {
       1 #( * N )*
    });
    assert_eq!(val, 0);
}
