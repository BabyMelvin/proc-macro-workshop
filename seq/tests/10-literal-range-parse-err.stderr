error: `invalid digit found in string`
       Only support `usize` type for now!
 --> tests/10-literal-range-parse-err.rs:2:30
  |
2 |     let val = seq::seq!(N in -1..1 {
  |                              ^^
